packages/braid-design-system/src/lib/components/ButtonIcon/ButtonIcon.docs.tsx

```diff
@@ -187,6 +188,53 @@ const docs: ComponentDocs = {
           </Stack>,
         ),
     },
+    {
+      label: 'Tooltip Placement',
+      background: 'surface',
+      description: (
+        <>
+          <Text>
+            Tooltips are positioned above the <Strong>ButtonIcon</Strong> by
```

Comment:
Might riff on this a little. Consumers dont provide a `tooltip` rather it is the `label` prop (we are presenting it as a tooltip).

Maybe something like:
> By default, the **label** is presented as a tooltip on top of the button. However, this can be configured via the **tooltipPlacement** prop, which accepts either **top** or **bottom**. See TooltipRenderer for more information.

Reply:
Yep good idea, just updated